docs/markdown/connecting-to-testnet.md

```diff
@@ -0,0 +1,61 @@
+# Connecting to an ILP Testnet
+
+This tutorial will help you start [`moneyd`](https://github.com/interledgerjs/moneyd) locally on your computer and 
+connect it to an [ILP router](https://github.com/interledgerjs/ilp-connector) on the testnet.
+
+This tutorial describes how to:
+
+1. Configure and start `moneyd` to connect to an ILP router on the testnet
+2. Send value to a [payment pointer](https://interledger.org/rfcs/0026-payment-pointers/) on the testnet
+
+## Before you begin
+For this tutorial, you will need `moneyd` and SPSP installed on your computer. You can learn how to install `moneyd` 
+and SPSP from the [Getting Started](getting-started.md) tutorial.
+
+NOTE: For this tutorial you do not need to use real cryptocurrency. Since you will be sending value through an 
+ILP router on the testnet, funds will be used from a [testnet faucet](https://xrpl.org/xrp-test-net-faucet.html) and have no real-world value.
+
+## Configuring moneyd
+NOTE: Before configuring `moneyd`, make sure to delete the `.moneyd.test.json` config file if it already exists on 
+your computer.
+
+To configure moneyd: 
+
+1. Open a terminal and use the following command:
+```shell
+moneyd xrp:configure --testnet --advanced
+```
+
+2. When prompted for the `BTP host of parent connector`, enter:
+```shell
+js.xpring.dev
+```
+
+3. Press enter for all other default options.
+
+## Starting moneyd
+After you’ve configured `moneyd`, run the following command to start it on your computer and connect it to an 
+ILP router on the testnet:
+```shell
+moneyd xrp:start --testnet
+```
+
+If everything goes well, you should see an output similar to this:
```

Comment:
Change the phrasing here to indicate if you don't see the connector ready log line, it won't work? (I kinda glanced over this and just assumed it connected, when it actually didn't)

Reply:
Usually, it is better to tell the users what you will see (and not what you don't see). I have modified the statement.
https://github.com/interledger/interledger.github.io/pull/108/commits/06659009ab81d44db7db3df5f318b5bae704a4ba#diff-95b449acc2e692458b91372a5b6e33e7R48

Reply:
Looks good. That one was probably just me not reading carefully enough :satisfied: